samples/gibbs_ensemble/run_for_several_kT.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+python=$(which python3)
+pypresso=../../build/pypresso
+
+if [[ ! -e $pypresso ]]
+then
+    echo Invalid path to pypresso script: $pypresso
+    exit 1
+fi
+
+# Run the simulations for the temperatures given as parameters
+for kT in $*
+do
+    for seed in 1 2 3 4 5
+    do
+
+        test -s "temp_$kT_$seed.dat.gz" ||
+            $pypresso run_sim.py $kT --log -steps 2000000 -seed $seed 2>&1 > "temp_$kT_seed_$seed.out" &
```

Comment:
On Bash 5.0.17, this code doesn't work, because `"temp_$kT_$seed.dat.gz"` expands variable `$kT_` to an empty string, when in reality we want to expand variable `$kT`. Please consider using the tool [koalaman/shellcheck](https://github.com/koalaman/shellcheck), it helped us a lot fixing buggy shell scripts in the past. Also, the pypresso variable needs to be in quotes in case it contains whitespaces, and the .dat.gz file had the wrong filename.
```suggestion
        test -s "temp_${kT}_seed_${seed}.dat.gz" ||
            "${pypresso}" run_sim.py ${kT} --log -steps 2000000 -seed ${seed} 2>&1 > "temp_${kT}_seed_${seed}.out" &
```

Reply:
Thank you for the shellcheck tool, I will keep it in mind for future scripts.